src/olympia/devhub/templates/devhub/new-landing/index.html

```diff
@@ -1 +1,70 @@
-New DevHub Landing
+<!DOCTYPE html>
+<html lang="{{ LANG }}" dir="{{ DIR }}">
+  <head>
+    <meta charset="utf-8">
+      {% if not settings.ENGAGE_ROBOTS %}
+        <meta name="robots" content="noindex">
+      {% endif %}
+    {% block extrameta %}{% endblock %}
+    <title>{% block title %}{{ dev_page_title() }}{% endblock %}</title>
+
+    <link rel="shortcut icon" type="image/x-icon"
+          href="{{ STATIC_URL }}img/favicon.ico">
+
+    {% block rss_feed %}{% endblock %}
```

Comment:
There is one feed (addons activity), it's not on the index page but might be relevant later.